### PR TITLE
61/my account addresses

### PR DESCRIPTION
**Pull request by qiqqq**

### CODE REVIEW

File: components/organisms/o-my-account-placeholder.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <SfTabs :open-tab="1" class="tab-orphan">
```
Comment (gibkigonzo) [inline components/organisms/o-my-account-placeholder.vue]: can you change this class into `o-my-account-placeholder`? or add div with class o-my-account-placeholder

File: components/organisms/o-my-account-shipping-details.vue [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+      >
+        <SfTab :title="$t('Change the address')">
+          <p class="message">
+            {{ $t('Keep your addresses and contact details updated.') }}
+          </p>
+          <div class="form">
+            <SfInput
+              v-model="editedAddress.firstname"
+              name="firstname"
+              label="First Name"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('First Name')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+                !$v.editedAddress.firstname.required
+                  ? $t('Field is required.')
+                  : $t('Name must have at least 2 letters.')
+              "
+              class="form__element form__element--half"
+            />
+            <SfInput
+              v-model="editedAddress.lastname"
+              name="lastname"
+              label="Last Name"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('Last Name')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+              required
+              :valid="!$v.editedAddress.lastname.$error"
+              :error-message="$t('Field is required.')"
+              class="form__element form__element--half form__element--half-even"
+            />
+            <SfInput
+              v-model="editedAddress.streetName"
+              name="streetName"
+              label="Street Name"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('Street Name')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+              required
+              :valid="!$v.editedAddress.streetName.$error"
+              :error-message="$t('Field is required.')"
+              class="form__element"
+            />
+            <SfInput
+              v-model="editedAddress.apartment"
+              name="apartment"
+              label="House/Apartment number"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('House/Apartment number')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+              required
+              :valid="!$v.editedAddress.apartment.$error"
+              :error-message="$t('Field is required.')"
+              class="form__element"
+            />
+            <SfInput
+              v-model="editedAddress.city"
+              name="city"
+              label="City"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('City')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+              required
+              :valid="!$v.editedAddress.city.$error"
+              :error-message="$t('Field is required.')"
+              class="form__element form__element--half"
+            />
+            <SfInput
+              v-model="editedAddress.state"
+              name="state"
+              label="State/Province"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('State/Province')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 8]
```diff
@@ -0,0 +1,429 @@
...
+              class="form__element form__element--half form__element--half-even"
+            />
+            <SfInput
+              v-model="editedAddress.postcode"
+              name="postcode"
+              label="Zip-code"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('Zip-code')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 9]
```diff
@@ -0,0 +1,429 @@
...
+                !$v.editedAddress.postcode.required
+                  ? $t('Field is required.')
+                  : $t('Zip-code must have at least 3 letters.')
+              "
+              class="form__element form__element--half"
+            />
+            <SfSelect
+              v-model="editedAddress.country"
+              name="country"
+              label="Country"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('Country')"
```

File: components/organisms/o-my-account-shipping-details.vue [hunk 10]
```diff
@@ -0,0 +1,429 @@
...
+                :key="country.code"
+                :value="country.name"
+              >
+                {{ country.name }}
+              </SfSelectOption>
+            </SfSelect>
+            <SfInput
+              v-model="editedAddress.telephone"
+              name="telephone"
+              label="Phone number"
```
Comment (psmyrek) [inline components/organisms/o-my-account-shipping-details.vue]:
```suggestion
              label="$t('Phone number')"
```

File: components/organisms/o-my-account-placeholder.vue (hunk 1 above)
Comment (qiqqq) [reply components/organisms/o-my-account-placeholder.vue]: you are right, added

File: components/organisms/o-my-account-shipping-details.vue (hunk 2 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 3 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 4 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 5 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 6 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 7 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 8 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 9 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

File: components/organisms/o-my-account-shipping-details.vue (hunk 10 above)
Comment (qiqqq) [reply components/organisms/o-my-account-shipping-details.vue]: Done

**Pull request merged**